core/dbt/task/build.py

```diff
@@ -0,0 +1,45 @@
+from .compile import CompileTask
+
+from .run import ModelRunner as run_model_runner
+from .snapshot import SnapshotRunner as snapshot_model_runner
+from .seed import SeedRunner as seed_runner
+from .test import TestRunner as test_runner
+
+from dbt.graph import ResourceTypeSelector
+from dbt.exceptions import InternalException
+from dbt.node_types import NodeType
+
+
+class BuildTask(CompileTask):
+    """
+    The Build task processes all assets of a given process and attempts to 'build'
+    them in an opinionated fashion.  Every resource type outlined in RUNNER_MAP
+    will be processed by the mapped runner class.
+
+    I.E. a resource of type Model is handled by the ModelRunner which is imported
+    as run_model_runner.
+    """
+
+    # TODO: is this list complete?
```

Comment:
Is this something we can answer now?

Reply:
I think so!  @jerco does this represent all of the node types that we would like to process when we run the build command?

Reply:
Yes! This is everything